src/gar.lua

```diff
@@ -838,6 +847,11 @@ function gar.updateGatewayStats(address, gateway, stats)
 	assert(stats.failedConsecutiveEpochs, "failedConsecutiveEpochs is required")
 	assert(stats.passedConsecutiveEpochs, "passedConsecutiveEpochs is required")
 
+	if stats.failedConsecutiveEpochs >= gar.getSettings().operators.failedEpochCountMax then
+		-- Ensure pruning takes place at the next opportunity
+		NextGatewaysPruneTimestamp = 0
+	end
```

Comment:
this happens on `distribute` which does not run automatically like `prune`. will not hurt, but the gateway won't be moved to `leaving` until the next `Tick` comes in

Reply:
Good to know. Will remove to avoid complexity.

Reply:
629ba1f